src/server/rest/v1/service/ChargingStationService.ts

```diff
@@ -65,12 +66,14 @@ export default class ChargingStationService {
         action: action
       });
     }
+    let site: Site = null;
     let siteArea: SiteArea = null;
     // Check the Site Area
     if (Utils.isComponentActiveFromToken(req.user, TenantComponents.ORGANIZATION) && filteredRequest.siteAreaID) {
       siteArea = await SiteAreaStorage.getSiteArea(req.user.tenantID, filteredRequest.siteAreaID);
       UtilsService.assertObjectExists(action, siteArea, `Site Area ID '${filteredRequest.siteAreaID}' does not exist.`,
         MODULE_NAME, 'handleUpdateChargingStationParams', req.user);
+      site = await SiteStorage.getSite(req.user.tenantID, siteArea.siteID);
```

Comment:
You can get the siteArea with his site above when calling SiteAreaStorage by providing the withSite flag.
This would avoid a second call to the DB.